src/Tribe/Models/Post_Types/Event.php

```diff
@@ -77,7 +77,7 @@ protected function build_properties( $filter ) {
 			$multiday = false;
 
 			if ( $all_day ) {
-				$start_end_diff = $start_date_object->diff( $end_date_object );
+				$start_end_diff = $start_date_object->diff( $end_date_object->add( $one_second ) );
```

Comment:
Why this?

Reply:
There's an issue here where if a multiday all day event starts on one day and ends on the next day, it doesn't count as multiday. For a concrete example, January 20, 2020 00:00:00 - January 21, 2020 23:59:59. This would return 1 day difference in the previous code, which would equate to a non multiday event. By adding one second, it fixes this to properly equate to the 2 days required.

Reply:
Ok, thanks for clarifying.